pkg/kms/creator.go

```diff
@@ -45,21 +42,11 @@ func NewServiceCreator(c *Config) ServiceCreator {
 		keystoreID := mux.Vars(req)[keystoreIDQueryParam]
 		keyURI := fmt.Sprintf(masterKeyURI, keystoreID)
 
-		b, err := cloneRequestBody(req)
-		if err != nil {
-			return nil, err
-		}
-
-		p := struct {
-			Passphrase string `json:"passphrase"`
-		}{}
+		// TODO(#22): Replace with split shares. Consider to use "Hub-Kms-Secret-A" and "Hub-Kms-Secret-B-Path"
+		// headers to pass a content of secret A and a location to secret B accordingly.
+		passphrase := req.Header.Get(passphraseHeader)
```

Comment:
please rename http header to `secret` as this eventually will be a secret share, the client doesn't need to know the type of this content, it's just a generic secret.

Reply:
done